src/style.css

```diff
@@ -0,0 +1,149 @@
+/* Global */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700&display=swap");
+
+:root {
+  --main-color: #060b35;
+  --secondary: #616070;
+  --blue-accent: #00acdc;
+  --white: #fff;
+  --gray: #e5e5e5;
+}
+
+.page {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+
+  background-color: var(--gray);
+  color: var(--main-color);
+}
+
+/* / Global */
+
+.reset {
+  margin: 0;
+}
+
+.card {
+  display: flex;
+  flex-wrap: wrap;
+  align-content: flex-start;
+  justify-content: center;
+
+  width: 200px;
+  min-height: 406px;
+
+  box-sizing: border-box;
+  background-color: var(--white);
+  border: 1px solid var(--gray);
+  border-radius: 5px;
+}
+
+.card__image {
+  margin-top: 32px;
+  width: 160px;
+  height: 134px;
+}
+
+.card__name {
+  width: 100%;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+
+  margin-top: 40px;
+  padding: 0 16px;
+}
+
+.card__id {
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+
+  flex-basis: 100%;
+
+  margin-top: 4px;
+  padding-left: 16px;
+
+  color: var(--secondary);
+}
+
+.card__price {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+
+  padding: 24px 16px 16px;
+}
+
+.card__price-name {
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+
+  color: var(--secondary);
+}
+
+.card__amount {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+}
+
+.card__buy-button {
+  width: 166px;
+  height: 40px;
+
+  text-align: center;
+  text-decoration: none;
+
+  font-size: 14px;
+  font-weight: 700;
+  line-height: 40px;
+  color: var(--white);
+  background-color: var(--blue-accent);
+  border: 1px solid transparent;
+  border-radius: 5px;
+  box-sizing: border-box;
+}
+
+.card__buy-button:hover {
+  color: var(--blue-accent);
+  background-color: var(--white);
+  border: 1px solid var(--blue-accent);
+}
+
+.stars-block {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+
+  padding: 16px 16px 0;
+}
+
+.stars-block__reviews-count {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+
+}
+
+/* Stars__block CSS from Stars Layout */
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star:nth-child(-n + 4) {
+  background-image: url(./images/star-active.svg);
+}
+
```

Comment:
use two different files for styles card and stars

Reply:
stars block styles moved to a separate file